### ISSUE TITLE
KCD Munich 2024

### ISSUE THREAD

**Issue by mkorbi** [cncf/kubernetes-community-days#504]
# KCD Organizer Checklist

### Event Details
* [x] Event Location: smartvillage Bogenhausen - Munich
* [x] Event Date and Time: July 1 & 2
* [x] Expected Number of Attendees 350
* [ ] Event cost per ticket, and if you will have early bird pricing or not

#### Event type:
* [ ] Virtual
* [ ] Hybrid
* [x] In person

#### Organizers Details (identify at least 3 organizers). Designate who will be the CFP admin, as well as the finanical officer:
* [x] Organizer 1: Max Körbächer, Liquid Reply, Founder, m.koerbaecher@reply.de, Max Körbächer
* [x] Organizer 2: Tom Uhlig, Liquid Reply, Consultant, t.uhlig@reply.de, Tom Uhlig
* [x] Organizer 3: Nico Meisenzahl, whiteduck, Cloud Solution Architect, nico.meisenzahl@whiteduck.de, Nico Meisenzahl
* [x] Organizer 4: Markus Sümmchen, whiteduck, Founder, markus.suemmchen@whiteduck.de, Markus Sümmchen
* [x] Organizer 5: Anela Avdibegovic, whiteduck, Marketing Manager, anela.avdibegovic@whiteduck.de, <>
* [x] Organizer 6: Suad Wolgram, whiteduck, Consultant, suad.wolgram@whiteduck.de, <>
* [x] Organizer 7: Mario-Leander Reimer, QAware, CTO, mlr@qaware.de, Mario-Leander Reimer

### Terms

* [x] No other Kubernetes Community Day event in same major city (check with past organizers)
* [x] Minimum of 3 organizers from 3 different organizations
* [x] Every organizer has an account on [Cloud Native Community Groups](https://community.cncf.io/)
* [x] Each organizer lives within the KCD city or just outside of it. For small countries, the organizers can live within the region, but must be able to easily travel to the KCD if it were in-person.
* [x] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer. NOTE: An employee of a member company, qualifies.
* [x] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [x] All organizer inquiries / communication for CNCF must be summarized via GitHub (comment under your KCD issue) for easy tracking
* [x] All organizers have taken [Inclusive Open Source Community Orientation (LFC102)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* [x] Agree to offer at least 3 complimentary tickets to increase diversity for the event
* [x] Share the sponsor prospectus with CNCF via your GitHub issue, and link it on your website.
* [x] Agree to ensure the final program schedule is diverse (e.g., not all speakers of one gender, one culture, or from 1 or 2 of the same company). [Here are some tips on how to create a diverse lineup.](https://docs.google.com/presentation/d/1fzT_BdavVKh3mnxxU-PBWyJq9JUfasKwHqekkbYVbw8/edit#slide=id.g56245ab439_0_106)
* [x] Agree to submit your final lineup to kcd@cncf.io for review, prior to publishing it
* [x] Agree to use Cloud Native Community Groups for registration, scheduling, and event website. If using a platform outside of Bevy, organizers must provide CNCF with a full list of attendees.
-- Regardless of your platform choice, please use [these questions](https://docs.google.com/spreadsheets/d/1OEoVQ8Y3eQmEEJeT2abhNOkYIkyk8nAVjebNFvKMTQQ/edit?usp=sharing) to create your registration form
* [x] Agree to use Sessionize for CFP and to close it at least 6 weeks before the event

### Getting started checklist for organizers

* [x] Schedule monthly check-ins with KCD Core Organizer(s) (kcd@cncf.io)
* [x] Minimum of 3 sponsors committed (not required for virtual only events)
- Liquid Reply
- whiteduck
- QAware

* [x] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
Fiscal sponsor that will accept payments: whiteduck
* [x] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel
* [x] Create a Twitter handle specific to your KCD event (except for countries like China and Korea where Twitter is not used) @KCDMunich

### Getting started checklist for CNCF:
* [x] Organizer email alias has been updated to only reflect above organizers: organizers-munich@kubernetescommunitydays.org 
* [x] Region created in Bevy
* [ ] Added to Bevy header
* [x] CFP created in Sessionize
* [ ] Marketing Plan shared

### Organizer Planning Timeline
* [x] Make a copy of [this timeline](https://docs.google.com/spreadsheets/d/1bvCiyyDut1seSnBE6pzVevcJkXLeWbxbncvhFsyY8PI/edit#gid=1736864170) based on whether your event is In-person, Hybrid, or Virtual and link your own spreadsheet here ().
* [x] To assure you have a diverse lineup, you are required to send a copy of your program to kcd@cncf.io before you publish it.

### Post event checklist for Organizer:
* [ ] Send CNCF event analytics
* [x] Schedule post mortem meeting
* [ ] Send CNCF videos to be uploaded to YouTube
* [ ] Link videos from YouTube to event page
* [ ] Post event survey sent out

### Post event checklist for CNCF:
* [ ] Removed from Bevy header
* [ ] CFP deactivated
* [ ] Upload videos to YouTube

**Comment by AudMonte01** [comment]
Looking into July dates. Kick-off meeting due to happen the week of November 27.

**Comment by AudMonte01** [comment]
Team missed Kick-off meeting. Postponed to December 4

**Comment by mkorbi** [comment]
[Sponsoring Prospectus](https://docs.google.com/presentation/d/1G7CbTjV023TC7kt7json3j0CDDN0FqVpm7Hz-pZh4cs/edit?usp=sharing)

**Comment by AudMonte01** [comment]
Excellent - I have added it to the KCD GitHub repo of open prospectuses which is also linked from cncf.io/kcds:
https://github.com/cncf/kubernetes-community-days/tree/main/sponsor-resources/open-prospectuses

**Comment by AudMonte01** [comment]
JANUARY 8 MEETING NOTES

KubeCon Nomination
* Team only just secured venue, and Paris is near enough to not need travel funding
* Will apply for another KubeCon this year

Sponsorship
* Fiscal sponsorship secured
* ApeFactory wants to come in as a Gold sponsor and have their logo on T-shirts

Action Items
* [ ] Team to update logo on website, and social media based on new KCD logo Audra shared in the organizer slack channel
* [ ] Audra to promote this CFP in KubeWeekly newsletter

**Comment by AudMonte01** [comment]
FEBRUARY 5 CNCF SYNC

Content
* 85 CFP Submissions so far
* Keynote speakers will come from CFP

Sponsors so far
* 3 or 4 silver sponsors
* 1 Gold
* 1 workshops

Marketing
* Think about using community.cncf.io for your meetups to better transition to the KCD, AND be backed by CNCF with [benefits](https://github.com/cncf/communitygroups/blob/main/README.md#benefits-of-joining-the-cncf-community-groups-program).
* [ ] Audra to plug Early Bird tickets (good through February) in KubeWeekly
* Reminder to take advantage of the  #socialmedia channel in the CNCF Slack Workspace and tag Jessie Adams-Shore to amplify from main CNCF account and Audra to do so via the Twitter (X) account
* [ ] Audra to add KCD Munich in drop-down menu on CNCF website

**Comment by AudMonte01** [comment]
MARCH 4 CNCF SYNC

Content
* CFP deadline is March 31
* 124 submissions so far (89 local speakers)
* 40 session slots

Sponsors
* 2 Golds, 6 Silver, 2 Community (5 new sponsors as of last week)
* 2 in the pipeline
* Would ideally secure 40-50k more for the same event experience as last year

Marketing
* 50 Early Tickets
* Promote CFP at community event Kiosk in Paris (Friday coffee break)
* [ ] Audra to point to sponsor prospectus in KubeWeekly newsletter

**Comment by AudMonte01** [comment]
RETROSPECTIVE

Positive Feedback
* The quality of the event was so high, that some people suggested their price should have been higher
* Red Hat 3-hour workshop was halfway full
* 1.5-hour workshops that were submitted via the CFP were well attended

Lessons Learned & Cons
* Little late signing some sponsor contracts
* Would be great to have a sponsor guide that people can use as a resource versus an email
* May raise prices for next year
* Registration for bowling party had more than showed up due to bulk tickets

Analytics
* Total registered was 438 (10-20 are 1-day passes) versus 406 checked-in (93%)
* Standard ticket price was 159 Euros (early bird was 109 Euros and late birds was 189 Euros)
* Percentage of non-men speakers (14% for those who showed up and 16% inclusive of the 1 no-show)
* Sponsorships: What to do with excess funds (should at least break even)
-- still waiting on final payments to come through
-- and final bill from the venue
-- venue gave a high-discount, but the event was about a 180k Euro event

Post-Event Marketing
* [ ] Videos are being worked on. Team will let Audra know once the playlist is ready, 
* [ ] Audra to share transparency report template in Slack channel
* [ ] 1 extra Credly badge for speakers. Team is working on badge. Team collecting un-conference speakers nice
* [ ] Blog post is TBD

Feedback
* It would be helpful for CNCF to have an insurance background if things go wrong (reduce blast radius)
* Financial support for newer ones, but have a fail-safe rescue system for advanced/legacy KCDs (emergency bucket from KCDs that use financial support)

**Comment by mkorbi** [comment]
@AudMonte01 In addition to the feedback, we should have CNCF as insurance in the background, however that looks.
We have event insurance (and you might ask other KCDs to have this too), but they usually have some limitations or directly cost a quadruple amount of money.

**Comment by AudMonte01** [comment]
Hello @mkorbi - checking in here on a few things:
* Transparency report status
* Videos and when you project these will be on YouTube?
* Will the team proceed with a blog post on how the KCD has evolved over time?

**Comment by nmeisenzahl** [comment]
Hi @AudMonte01 we have published the session recordings yesterday: https://www.youtube.com/@KCDMunich

**Comment by AudMonte01** [comment]
Thank you, @nmeisenzahl - I have made a copy of this playlist to the CNCF YouTube channel to amplify to our subscribers.

Please note, I found some duplicates on your playlist:
https://youtu.be/bSuYqi1A2jM?si=4eUd_S1v7Qnf2bNm
https://youtu.be/cCFYrZpwS7s?si=aodrWhn6D95LWW-O
https://youtu.be/nyxKd96TQ24?si=RKlIrJICZffInO2a
https://youtu.be/o3MYeUWhETo?si=dV6Y3JeUGCh-4kF7
https://youtu.be/oCFNrXWN-HE?si=Vvju-RyjTR7GBKu4
https://youtu.be/cCFYrZpwS7s?si=ujYGbV7oVaCjLgXB (showed up 4 times in the playlist)

And there may be more, but these are what I caught.
